data/trie/trieStorageManager.go

```diff
@@ -352,9 +352,6 @@ func (tsm *trieStorageManager) writeOnChan(entry snapshotsQueueEntry) {
 	select {
 	case tsm.snapshotReq <- entry:
 		return
-	default:
```

Comment:
Maybe we should let this print and change the value of SnapshotsBufferLen from config.toml to something more reasonable as 1000 and if something goes wrong and these messages would not be consumed we could be triggered by this print.

Reply:
there is no need for this default. 1000 is not enough. you can have much more data tries in the same shard.

Reply:
ok